### DISCUSSION TITLE
Bungee & Velocity Support

### DISCUSSION THREAD

**Discussion by ghost** [QuickShop-Community/QuickShop-Hikari#120] [Ideas]
In the current version, if you buy items in the store on one server, the purchase prompt will not be displayed immediately on another server. You must join that server to see it. I hope that in later versions, BC/VC synchronization can be supported. Purchase prompts can be seen on any server
(I used machine translate because i don't speak English)

**Comment by Ghost-chu** [comment]
> In the current version, if you buy items in the store on one server, the purchase prompt will not be displayed immediately on another server. You must join that server to see it. I hope that in later versions, BC/VC synchronization can be supported. Purchase prompts can be seen on any server (I used machine translate because i don't speak English)

have you enabled cross server message feature?

```yaml
#Should we try to send cross-server message for the player?
bungee-cross-server-msg: true
```

**Reply by Ghost-chu** [reply]
If it doesn't work, please open a issue on Issue Tracker :)

**Reply by ghost** [reply]
Ok, maybe I'm blind and didn't see this option, can quickshop use one database connection for multiple servers?

**Reply by Ghost-chu** [reply]
> Ok, maybe I'm blind and didn't see this option, can quickshop use one database connection for multiple servers?

Just change the `database.prefix` to different prefixes to spilt the database tables.

**Reply by ghost** [reply]
ok, thanks
